src/main/java/gov/cms/madie/models/measure/TestCase.java

```diff
@@ -65,6 +68,9 @@ public class TestCase implements Serializable, Cloneable {
 
   @Valid private List<TestCaseGroupPopulation> groupPopulations;
 
+  private String testCaseValidationStatus;
+  private String testCaseValidationMessage;
```

Comment:
@nmorasb @RohitKandimalla- are we going to capture just the validation message or the location in the json as well? maybe depends on usecase? if we want to provide scrollto feature upon clicking the error, we might need location of the error.

Reply:
okay, we already have one-
`@Transient private HapiOperationOutcome hapiOperationOutcome;`
Should we just repurpose it? maybe we can rename it or leave it the way it is.

Reply:
@adongare @nmorasb The card specifies to have a validation status field of enum type and a validation message. I don't think we can repurpose it at this point, because hapiOperationOutcome is still used in both frontend and backend. And I don't know what value to map into the status enum.

Reply:
@RohitKandimalla Ok thanks, I'll remove testCaseValidationMessage then.

Reply:
Are we going to store the validation messages?  if yes- hapioperationoutcoe is transient. we will have to remove the transient annotation so that it can be persisted.

Reply:
@adongare Yeah I think we are storing the validation message, removed transient annotation.